SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialWorkerFlags.cpp

```diff
@@ -13,25 +13,20 @@ bool USpatialWorkerFlags::GetWorkerFlag(const FString& InFlagName, FString& OutF
 	return false;
 }
 
-void USpatialWorkerFlags::ApplyWorkerFlagUpdate(const Worker_FlagUpdateOp& Op)
+void USpatialWorkerFlags::SetWorkerFlag(const FString& InFlagName, const FString& OutFlagValue)
```

Comment:
I'd make these by value instead of const ref as you are going to assume ownership of the values either way. And in practice you are going to be calling this from a temporary most of the time.

Reply:
or make a rvalue reference overload : p. (Nah that would be overkill)